monai/auto3dseg/algo_gen.py

```diff
@@ -44,6 +44,10 @@ def get_score(self, *args, **kwargs):
         """Returns the model quality measurement based on training and validation datasets."""
         pass
 
+    def get_output_path(self, *args, **kwargs):
+        """Returns the model output paths"""
```

Comment:
model or algo? would be great to make the terminology consistent...

Reply:
It is algo, not model. Thanks!